src/lib.rs

```diff
@@ -596,6 +596,12 @@ pub use oauth2::{
     TokenResponse as OAuth2TokenResponse, TokenType, TokenUrl,
 };
 
+#[cfg(feature = "backchannel-logout")]
+mod logout_token;
+#[cfg(feature = "backchannel-logout")]
+#[doc(inline)]
+pub use logout_token::*;
```

Comment:
This might actually be not such a good idea, I'll remove that. Otherwise, it might get confusing because of stuff like `Identifier`